### PR TITLE
create seeds and connection, data files

### PR DESCRIPTION
**Pull request by aakirahin**

### CODE REVIEW

File: db/data/test-data/ducks.js [hunk 1]
```diff
@@ -0,0 +1,386 @@
+module.exports = [
```
Comment (ch3wyb01) [inline db/data/test-data/ducks.js:1]: Need to make sure that finder_id and maker_id aren't the same

File: db/seeds/seed.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+const db = require("../connection.js");
```
Comment (ch3wyb01) [inline db/seeds/seed.js:1]: We can change this to async/await for readability

File: db/seeds/seed.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          user.first_name,
+          user.last_name,
+          user.password,
+          user.email,
+          user.profile_pic,
+        ];
+      });
+      const usersQuery = format(
+        `INSERT INTO users (user_name, first_name, last_name, password, email, profile_pic)
+        VALUES %L RETURNING *;`,
```
Comment (ch3wyb01) [inline db/seeds/seed.js:55]: We don't need 'RETURNING *'

File: db/seeds/seed.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        maker_id,
+        finder_id,
+        location_placed_lat,
+        location_placed_lng,
+        location_found_lat,
+        location_found_lng,
+        clue,
+        image,
+        comments)
+        VALUES %L RETURNING *;`,
```
Comment (ch3wyb01) [inline db/seeds/seed.js:87]: We don't need 'RETURNING *'

File: db/data/development-data/ducks.js [hunk 5]
```diff
@@ -0,0 +1,3816 @@
+module.exports = [
+  {
```
Comment (ch3wyb01) [inline db/data/development-data/ducks.js]: Need to make sure that finder_id and maker_id aren't the same

File: db/connection.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+          rejectUnauthorized: false,
+        },
+      }
+    : {};
+
+require("dotenv").config({
+  path: `${__dirname}/.env.${ENV}`,
+});
+
+console.log(`Connected to ${process.env.PGDATABASE}`);
```
Comment (ch3wyb01) [inline db/connection.js:18]: remove console.log

File: db/seeds/run-seed.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const devData = require("../data/development-data/index.js");
+const seed = require("./seed.js");
+const db = require("../connection.js");
+
+const runSeed = () => {
+  console.log("Running seed");
```
Comment (ch3wyb01) [inline db/seeds/run-seed.js:6]: Remove console.log

File: db/seeds/seed.js [hunk 8]
```diff
@@ -0,0 +1,94 @@
+const db = require("../connection.js");
+const format = require("pg-format");
+
+const seed = (data) => {
+  const { duckData, userData } = data;
+  console.log("In seed");
```
Comment (ch3wyb01) [inline db/seeds/seed.js:6]: Remove console.log

**Review by ch3wyb01 — CHANGES_REQUESTED** [review]

**Review by Web-Dev-Rich — APPROVED** [review]

**Review by Web-Dev-Rich — APPROVED** [review]

**Review by Web-Dev-Rich — APPROVED** [review]

**Review by Web-Dev-Rich — APPROVED** [review]

**Review by Web-Dev-Rich — APPROVED** [review]

**Review by ch3wyb01 — APPROVED** [review]

**Pull request merged**